packages/cli/src/pages/api/lists/[id]/members/[memberId].ts

```diff
@@ -0,0 +1,115 @@
+import { NextApiRequest, NextApiResponse } from "next";
+import {
+  ResError,
+  validate,
+  validationErrorResponse,
+} from "src/util/api/validate";
+import { validateMemberStatusInList } from "src/util/api/validateMemberStatusInList";
+import { withSessionAPIRoute } from "src/util/session";
+import prisma from "../../../../../../prisma";
+
+interface Data {
+  error?: any;
+  member?: any;
+}
+
+async function handlePatchListMember(
+  listId: string,
+  memberId: string,
+  req: NextApiRequest,
+  res: NextApiResponse<Data>
+) {
+  const status = req.body.status;
+
+  const validatedMemberStatusInList = validateMemberStatusInList(status);
+
+  if (validatedMemberStatusInList.hasError) {
+    return validationErrorResponse(validatedMemberStatusInList, res);
+  }
+
+  // todo: why does this need to be updateMany?
```

Comment:
I think it's because there is not a unique constraint on listId + email

Schema needs `@@unique([listId, email])` for this to work

Reply:
✔️ I think it's good to add that index so I added it, but you still have to use updateMany

Reply:
ftfy